movieapp/src/hooks/MovieContext.tsx

```diff
@@ -30,10 +41,21 @@ const useNewMovies = (): any => {
   const { newMovies, setNewMovies } = useContext(NewMoviesContext);
 
   const deleteNewMovies = (id: string): void => {
+    const requestOptions = {
+      method: 'DELETE',
+      headers: { 'Content-Type': 'application/json' }
+    };
+    fetch('http://localhost:4000/movies/' + id, requestOptions);
     setNewMovies(newMovies.movies.filter((el: { id: any }) => el.id !== id));
   };
 
   const addNewMovies = (movieToAdd: MovieType): any => {
+    const requestOptions = {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
DRY - don't repeat yourself.  

create util or wrapper that will provide api and encapsulate logic with providing same headers, methods, fetch, body

and u'll use it as `apiPost(url, body)`